source/instructors/setupcourses/course-llm.rst

```diff
@@ -0,0 +1,42 @@
+.. meta::
+   :description: Instructions for managing LLM usage in your course
+
+
+.. _course_llm:
+
+Large Language Model
+====================
+
+Once :ref:`LLM API keys <org_llm>` have been added to your account, the course can use the keys set
+
+    .. image:: /img/llm_course_settings.png
+       :alt: LLMs Course settings
+
+When enabled you can set limits for the students in the course
+
+
+Reset LLM Keys
+**************
+
+    .. image:: /img/llm_course_reset.png
+       :alt: LLMs Course Reset
+
+If students exceed any of the limits set above, they will show in the **Reset LLM Keys** area and the teacher/instructors can reset the keys for them to continue working.  
+
+If a student exceeds more than one limit at same time, they will only show once and reetting will reset them for all exceeded limits
+
+.. Note:: Students WILL need to restart their box after being limits are reset (**Project>Restart Box**) but if you hide the top menu then teachers should restart the box for them - open the student assignment where they will see the main menu and can **Project>Restart Box** before letting the student know they can continue
```

Comment:
after being limits are reset -> after limits are reset. If the top menu has been hidden for students then instructors will need to restart their box for them. Open the student assignment and select **Project>Restart Box** before letting the student know they can continue.